### DISCUSSION TITLE
Retevis RT3S Issue: Expected reference to a group call digital contact.

### DISCUSSION THREAD

**Discussion by wp4vw** [hmatuschek/qdmr#273] [General]
Hello,

Please advise how to fix this issue.

73's
Milton WP4VW

**Comment by hmatuschek** [comment]
You can't you should only add group-calls to the group lists. Some radios allow to add private calls to these lists, but to my knowledge, the OpenGD77 firmware is the only one, that allows that.

**Reply by jmelvi200** [reply]
Running into the same on TYT MD-UV380G. The only alteration that I made after downloading from the radio was changing the frequency of one channel, but I encountered this message when I attempted to re-upload to it. So assuming what you say is the case, the radio doesn't seem to care about private calls being in the group lists...just qdmr. At least on these radios anyway.

**Reply by allesand** [reply]
@WP4VW , @jmelvi200 :

Since this has come up a number of times now (also in #332 and https://github.com/hmatuschek/qdmr/discussions/219) : an "RX group list" should contain only groups, should it? I can see why, when using OpenGD77, some put PCs in the group list (to be able to "quick access" them with right/left key for *making* calls to those IDs), but that indeed breaks the "one config for all radios" concept of qdmr - which is very cool, though. Besides the "quick access with OpenGD77", why should there be a PC in an RX groups list? Just trying to figure out whether things are maybe used differently on different radios.
